openedx/core/djangoapps/programs/utils.py

```diff
@@ -113,6 +114,16 @@ def get_programs_for_credentials(user, programs_credentials):
     return certificate_programs
 
 
+def get_display_category(program):
+    display_candidate = None
+    if program and program.get('category', None):
+        if program.get('category', None) == 'xseries':
```

Comment:
nit: `None` is the default second argument so it's not necessary to pass it here.